app/ResolveRoute.js

```diff
@@ -7,8 +7,9 @@ export const routeRegex = {
     CategoryFilters: /^\/(hot|created|trending|active|promoted)\/?$/ig,
     PostNoCategory: /^\/(@[\w\.\d-]+)\/([\w\d-]+)/,
     Post: /^\/([\w\d\-\/]+)\/(\@[\w\d\.-]+)\/([\w\d-]+)\/?($|\?)/,
+    PostJson: /^\/([\w\d\-\/]+)\/(\@[\w\d\.-]+)\/([\w\d-]+)(\.json)$/,
     UserJson: /^\/(@[\w\.jso\d-]+)(\.json)$/,
```

Comment:
Not sure, seems it might of got added in your commit here https://github.com/steemit/steemit.com/pull/1122/commits/6c55b10b45fe909a12a68ddcf07e5384969146f3